# Description.msg: A description for all objects on a table (for
# hands-free PbD system).

# Fields
# ======================================================================
string[] object_names
string[] descriptions
